scripts/raspbian-install.sh

```
@@ -159,27 +183,47 @@ confirm_installation() {
+        err "Could not find wifi-connect UI package"
+    fi
+
+    say "Downloading and extracting $_binary_url..."
 
     _download_dir=$(ensure mktemp -d)
 
-    ensure curl -Ls "$_arch_url" | tar -xz -C "$_download_dir"
+    # Download and extract the binary
+    ensure curl -Ls "$_binary_url" | tar -xz -C "$_download_dir"
 
+    ensure sudo mkdir -p $INSTALL_BIN_DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)